packages/array-range/index.d.ts

```diff
@@ -0,0 +1,3 @@
+declare function range(start: number, end: number|null, step?: number|null): number[];
+declare function range(end: number, step?: number|null): number[];
```

Comment:
I don't think this works because if there are only two arguments it assumes they are `start` and `end`. Can probably just drop this line entirely

Reply:
Ah, here I was intending to capture the `range(5) -> [0,1,2,3,4]` case. But indeed, `step` can't be used there so just `range(end: number) : number[]` would be more correct.. 

This is mostly just to help out the autocomplete but I'm fine with just omitting the second line completely if you'd prefer.

Reply:
I see. Yeah probably fine to leave it as you have it then.